chart/elastalert2/values.yaml

```diff
@@ -89,6 +89,9 @@ elasticsearch:
 # Optional env variables for the pod
 optEnv: []
 
+## Enable an Specify container in extraContainers. This can be used to define a sidecar for updating rules
```

Comment:
Please correct grammar in this comment. Ex: `Specify optional additional containers to run alongside the Elastalert2 container."